lib/rubocop/cop/rails/pluralization_grammar.rb

```diff
@@ -0,0 +1,93 @@
+# encoding: utf-8
+
+module RuboCop
+  module Cop
+    module Rails
+      # This cop checks for correct grammar when using active support's
+      # core extensions to the numeric classes
+      #
+      # @example
+      #   # bad
+      #   3.day.ago
+      #
+      #   # good
+      #   45.minutes.from_now
+      class PluralizationGrammar < Cop
+        DURATION_METHODS = [:second, :seconds,
```

Comment:
I don't know how I feel about this being an array. It works well for the flat lookup of is this method one we care about, but having a mapping of the singular method to the plural method (and vice versa) feels like it would be better than adding or removing an 's' to singularize and pluralize.